app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SessionDetailActivity.java

```diff
@@ -13,10 +14,14 @@
 public class SessionDetailActivity extends BaseActivity {
 
     private static final String EXTRA_SESSION_ID = "session_id";
+    private static final String EXTRA_PARENT_NAME = "parent_name";
+
+    private String parentName;
 
     public static Intent createIntent(@NonNull Context context, int sessionId) {
         Intent intent = new Intent(context, SessionDetailActivity.class);
         intent.putExtra(EXTRA_SESSION_ID, sessionId);
+        intent.putExtra(EXTRA_PARENT_NAME, context.getClass().getCanonicalName());
```

Comment:
Ah, I found `createIntent` is called from non-activity, BroadcastReceiver.
Therefore my idea is not good.